bot/exts/help_channels/_cog.py

```diff
@@ -461,3 +487,25 @@ async def wait_for_dormant_channel(self) -> discord.TextChannel:
         self.queue_tasks.remove(task)
 
         return channel
+
+    async def update_available_help_channels(self) -> None:
+        """Updates the dynamic message within #how-to-get-help for available help channels."""
+        if not self.available_help_channels:
+            available_channels_category = await channel_utils.try_get_channel(Categories.help_available)
+            self.available_help_channels = set(
+                c.id for c in available_channels_category.channels if 'help-' in c.name
+            )
+
+        available_channels = AVAILABLE_HELP_CHANNELS.format(
+            available=', '.join(f"<#{c}>" for c in self.available_help_channels) or None
+        )
+
+        if self.how_to_get_help is None:
+            self.how_to_get_help = await channel_utils.try_get_channel(Channels.how_to_get_help)
```

Comment:
Could you add a check for `how_to_get_help` being an instance of TextChannel? GuildChannels could be voice or category channels, which don't all impleemnt fetch_message, send, and last_message.

Perhaps we should log a warning and abort if the channel is not of the correct type.

Reply:
I don't think this is necessary since there's no plans of having a voice channel anywhere near the available help channels.